src/main/java/teammates/ui/template/FeedbackResponseCommentRow.java

```diff
@@ -47,7 +47,7 @@ public FeedbackResponseComment(FeedbackResponseCommentAttributes frc, String giv
     }
 
     // for editing / deleting comments
-    public FeedbackResponseComment(FeedbackResponseCommentAttributes frc, String giverDisplay,
+    public FeedbackResponseCommentRow(FeedbackResponseCommentAttributes frc, String giverDisplay,
             String giverName, String recipientName, String showCommentToString,
```

Comment:
Would it be better to align such lines to below & after the bracket?
E.g: 
![image](https://cloud.githubusercontent.com/assets/12380232/15574100/05727444-237e-11e6-9904-921929a4cac4.png)